### PR TITLE
Support timeout and retries options in %packages section

### PR DESCRIPTION
**Pull request by poncovka**
Enable to set dnf's timeout and retries with a kickstart file.

(cherry-picked from a commit edd8c2d)

### CODE REVIEW

File: pyanaconda/payload/dnfpayload.py [hunk 1]
```diff
@@ -557,6 +563,8 @@ def _configure(self):
         dnf_logger = get_dnf_logger()
         dnf_logger.setLevel(dnf.logging.DDEBUG)
 
+        log.debug("Dnf configuration:\n%s", conf.dump())
```
Comment (jkonecny12) [inline pyanaconda/payload/dnfpayload.py]: Please use `format` to add `core.dump()` to the text.

Comment (poncovka) [reply pyanaconda/payload/dnfpayload.py]: Fixed.

**Comment by jkonecny12** [comment]
Ups:

``W1202(logging-format-interpolation):566,18: DNFPayload._configure: Use % formatting in logging functions and pass the % parameters as arguments``

This needs to be solved somehow. Not in this PR but before merging it for sure.

**Comment by jkonecny12** [comment]
Based on [this](https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation), I guess this should be exception to format rule.

**Comment by poncovka** [comment]
I agree, it should be an exception.

**Comment by InStallker** [comment]
test

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Pull request merged**